protocol-definitions/MC.yaml

```diff
@@ -741,3 +742,55 @@ methods:
           since: 2.0
           doc: |
             List of events.
+  - id: 25
+    name: getCPMembers
+    since: 2.0
+    doc: |
+      Returns the current list of CP members.
+    request:
+      retryable: true
+      partitionIdentifier: -1
+    response:
+      params:
+        - name: cpMembers
+          type: EntryList_String_String
```

Comment:
UUID has its own type in the protocol. We should not represent them as string.

Reply:
Same applies to `Address`. But that means that I need to add at least one more custom data type, e.g. `EntryList_UUID_Address`. Does it sound OK to you?

Reply:
It is ok.It is better to have correct types.

Reply:
Thanks. Going to fix it a bit later and ask you for another review round.

Reply:
Done.